clang/examples/PrintFunctionNames/PrintFunctionNames.cpp

```diff
@@ -72,7 +72,7 @@ class PrintFunctionsConsumer : public ASTConsumer {
           *sema.LateParsedTemplateMap.find(FD)->second;
       sema.LateTemplateParser(sema.OpaqueParser, LPT);
       llvm::errs() << "late-parsed-decl: \"" << FD->getNameAsString() << "\"\n";
-    }   
+    }
```

Comment:
Unrelated whitespace change?

Reply:
I need to make some changes in Clang for testing purposes. This will be removed before merging.